docs/bestPractices.md

```diff
@@ -0,0 +1,43 @@
+# Custom Element Best Practices
+
```

Comment:
I feel like there be some kind of header here. Something that references the actual scope of the document. Maybe a tl;dr kind of thing?